ops/mainnet/prod/core/config.tf

```diff
@@ -211,9 +223,33 @@ locals {
     environment  = var.stage
     databaseUrl  = "postgresql://${var.postgres_user}:${var.postgres_password}@db.mainnet.connext.ninja:5432/connext"
     healthUrls = {
-      prover = "https://betteruptime.com/api/v1/heartbeat/${var.lighthouse_prover_heartbeat}"
+      prover    = "https://betteruptime.com/api/v1/heartbeat/${var.lighthouse_prover_heartbeat}"
       processor = "https://betteruptime.com/api/v1/heartbeat/${var.lighthouse_processor_heartbeat}"
     }
     hubDomain = "6648936"
   })
+
+  local_relayer_config = jsonencode({
+    redis = {
+      host = module.sequencer_cache.redis_instance_address,
```

Comment:
Same redis as sequencer, sure here? Don't know the architecture, just pointing it out